back/app/services/input_ui_schema_generator_service.rb

```diff
@@ -48,9 +48,14 @@ def visit_page(field)
   def default_options(field)
     defaults = {
       isAdminField: admin_field?(field),
-      hasRule: field.logic?,
-      answer_visible_to: field.answer_visible_to
+      hasRule: field.logic?
     }
+    if field.resource # Required as the budget and author fields added don't have a resource
```

Comment:
Shouldn't we set answer_visible_to to the author and budget fields in the schema as well?

Reply:
Yes probably - are they both public?